src/assets/stylesheets/layout/_topics.scss

```diff
@@ -38,7 +34,6 @@
         font-weight: bold;
         margin-bottom: 0.5rem;
         font-size: 0.8rem;
-        white-space: nowrap;
```

Comment:
This was forcing each card to have at least the width of the title. But it also caused some cards to be wider than the others, which looked worse than having the titles wrapping across two lines.